includes.php

```diff
@@ -27,28 +27,38 @@ function hookpress_get_fields( $type ) {
 	}
 
 	// if it's a POST, we have a URL for it as well.
-	if ($type == 'POST' || $type == 'PARENT_POST')
+	if ($type == 'POST' || $type == 'PARENT_POST') {
+
 		$fields[] = 'post_url';
 
+		$meta_keys = $wpdb->get_col("select distinct(meta_key) from $wpdb->postmeta");
```

Comment:
What is the purpose here? Adding separate default columns might be something to put in a separate patch.